src/app/containers/PageHandlers/withOptimizelyProvider/index.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as optimizelyReactSdk from '@optimizely/react-sdk';
+import { render } from '@testing-library/react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import withOptimizelyProvider from '.';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+describe('withOptimizelyProvider HOC', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const optimizelyProviderSpy = jest.spyOn(
```

Comment:
Pretty sure this has no effect
```suggestion
  const optimizelyProviderSpy = jest.spyOn(
```

Reply:
I think in this test you're right, it doesn't because we're only mocking for a single test so can delete and push. Thanks!

Reply:
Done